### DISCUSSION TITLE
vendor/laravel/nova/resources/css/app.css fails to build

### DISCUSSION THREAD

**Discussion by MenC-Didam** [laravel/nova-issues#4802] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4.13
- PHP Version: 8.1
- Operating System and Version: Docker Alpine / Homestead

### Description:

ERROR in ./resources/css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
SyntaxError
(1153:1) /home/vagrant/code/.vapor/build/app/vendor/laravel/nova/resources/css/app.css @layer utilities is used but no matching @tailwind utilities directive is present.

### Detailed steps to reproduce the issue on a fresh Nova installation:

run npm / yarn run prod  from laravel/nova root.

Never had an issue until the most recent nova update.

**Comment by crynobone** [comment, marked as answer]
We run npm ci and npm run dev/prod on each CI before running dusk tests without failure.

**Reply by mateusgalasso** [reply]
I have the same error.
Some solution?

**Reply by mateusgalasso** [reply]
Just commented out "tailwind css: {}," in "postcss.config.js" file. Problem solved.

**Comment by mateusgalasso** [comment]
Just commented out "tailwind css: {}," in "postcss.config.js" file. Problem solved.
